core/app/models/spree/order_promotion.rb

```diff
@@ -2,5 +2,15 @@ module Spree
   class OrderPromotion < Spree::Base
     belongs_to :order, class_name: 'Spree::Order'
     belongs_to :promotion, class_name: 'Spree::Promotion'
+
+    delegate :name, :description, to: :promotion
+    delegate :currency, to: :order
+
+    extend Spree::DisplayMoney
```

Comment:
Ruby style guide suggests to move all `include` and `extends` to the top of the class definitions (https://github.com/rubocop-hq/ruby-style-guide#consistent-classes).

Reply:
@piotrleniec-spark we're doing it differently for money - we like to keep it close-by (the extend and `money_methods` declaration)